install/kubernetes/helm/meshery-operator/Chart.yaml

```diff
@@ -21,12 +21,12 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.6.0
+version: 0.6.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 # TODO: Need the confirm the image tag.
-# appVersion: 0.6.0
+# appVersion: 0.6.1
```

Comment:
@leecalcote I updated the following charts version:
- operator : 0.6.0  -> 0.6.1
- appVersion: 0.6.0 -> 0.6.1
- meshery: 0.5.0 -> 0.6.0
is this okay?

Reply:
Operator is already at chart version 0.6.0. 
Move Meshery to chart version 0.6.0
And do not uncomment or add appVersion. appVersion is added automatically by git tag in the release workflow.
Hope this clarifies.

Reply:
Okay, thanks a lot! done!